src/index.html

```diff
@@ -3,11 +3,359 @@
+            alt="blue-rectangle"
+            class="rectangle-right">
+          </div>
+          <div class="testimonials__card--text">
+            <img src="images/testimonials/quote.png" alt="quote">
+            <span class="testimonials__card--text-content">AIR’s ideas are refreshing and out of the box. Authentic team that focuses on the important path of the brand.</span>
+            <p class="testimonials__card--text-author">Tal Gilad</p>
+            <span class="testimonials__card--text-company">Teach for America</span>
+          </div>
+        </div>
+        <div class="testimonials__card">
+          <div class="testimonials__card__image-wrapper">
+            <img
+            src="images/testimonials/Rectangle 11.png"
+            alt="blue-rectangle"
+            class="rectangle-left">
+            <img
+            src="images/testimonials/testimonial-ava-2.jpg"
+            alt="image2"
+            class="testimonials__image">
+            <img
+            src="images/testimonials/Rectangle 11.png"
+            alt="blue-rectangle"
+            class="rectangle-right">
+          </div>
+          <div class="testimonials__card--text">
+            <img src="images/testimonials/quote.png" alt="quote">
+            <span class="testimonials__card--text-content">AIR is an exceptional agency that leads with creative talent, first-class account servicing.</span>
+            <p class="testimonials__card--text-author">Azadeh Hawkins</p>
+            <span class="testimonials__card--text-company">Hawkins Consulting</span>
+          </div>
+        </div>
+        <div class="testimonials__card">
+          <div class="testimonials__card__image-wrapper">
+            <img
+            src="images/testimonials/Rectangle 11.png"
+            alt="blue-rectangle"
+            class="rectangle-left">
+            <img
+            src="images/testimonials/testimonial-ava-3.jpg"
+            alt="image3"
+            class="testimonials__image">
+            <img
+            src="images/testimonials/Rectangle 11.png"
+            alt="blue-rectangle"
+            class="rectangle-right">
+          </div>
+          <div class="testimonials__card--text">
+            <img src="images/testimonials/quote.png" alt="quote">
+            <span class="testimonials__card--text-content">AIR raises the agency bar to stratospheric heights on both creative output and client service.</span>
+            <p class="testimonials__card--text-author">Michel Grover</p>
+            <span class="testimonials__card--text-company">Hulu</span>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="cta">
+      <div class="cta__text">
+        <h2>Vision, Passion, Results</h2>
+        <span>We are sure that first-rate job is possible only if all three components are united.</span>
+        <button class="btn">
+          <a href="#hire-us">Apply</a>
+        </button>
```

Comment:
The entire button needs to link to hire-us; currently the redirect only works if the user clicks exactly on the text and not on the button! [Checklist #15](https://github.com/mate-academy/layout_dia#:~:text=In%20the%20Vision%2C%20Passion%2C%20Results%20section%2C%20the%20%E2%80%9CApply%E2%80%9D%20button%20also%20should%20lead%20to%20the%20contact%20form)